packages/tools/devtools/devtools-core/src/ContainerDevtools.ts

```diff
@@ -549,4 +568,11 @@ export class ContainerDevtools implements IContainerDevtools, HasContainerKey {
 	): Promise<FluidObjectNode | undefined> {
 		return this.dataVisualizer?.render(fluidObjectId) ?? undefined;
 	}
+
+	/**
+	 * Begins the process of changing data inside a DDS by using {@link Edit}
```

Comment:
Nit: We don't generally document functions that return promises as "begins doing x". The fact that the method returns a promise is considered enough of a signal that it's async. I would recommend just documenting this as "Applies the edit..."